pkg/reconciler/common/install.go

```diff
@@ -52,7 +52,6 @@ func Install(ctx context.Context, manifest *mf.Manifest, instance v1alpha1.KComp
 		return fmt.Errorf("failed to apply non rbac manifest: %w", err)
 	}
 	status.MarkInstallSucceeded()
-	status.SetVersion(TargetVersion(instance))
```

Comment:
I understand why we can't do this here, but I still think it's important to denote somewhere in the CR exactly which version the operator installed. The current implementations of `MarkInstallSucceeded` call `MarkTrue` but let's have them call `MarkTrueWithReason` instead and include the `TargetVersion` in that reason.